fintech-examples/fintech-server/src/test/java/de/adorsys/opba/fintech/server/FinTechServerTests.java

```diff
@@ -0,0 +1,261 @@
+                        .header("X-Request-ID", UUID.randomUUID().toString())
+                        .header("X-XSRF-TOKEN", xsrfToken)
+                        .param("keyword", keyword)
+                        .param("start", start.toString())
+                        .param("max", max.toString()))
+                .andDo(print())
+                .andExpect(MockMvcResultMatchers.status().isOk())
+                .andReturn();
+
+        return new JSONObject(mvcResult.getResponse().getContentAsString()).getJSONArray("bankDescriptor").getJSONObject(0).get("uuid").toString();
+    }
+
+    @Test
+    @SneakyThrows
+    public void bankSearchUnAuthorized() {
+        String xsrfToken = "unvalidtoken";
+        this.mvc
+                .perform(MockMvcRequestBuilders.get(FIN_TECH_BANK_SEARCH_URL)
+                        .header("X-Request-ID", UUID.randomUUID().toString())
+                        .header("X-XSRF-TOKEN", xsrfToken)
+                        .param("keyword", "affe")
+                        .param("start", "0")
+                        .param("max", "2"))
+                .andDo(print())
+                .andExpect(MockMvcResultMatchers.status().isUnauthorized());
+    }
+
+    /**
+     * Tnis test does
+     * authorization
+     * bankSearch
+     * bankProfile
+     */
+    @Test
+    @SneakyThrows
+    public void bankProfileAuthorized() {
+        bankSearchService.setTppBankSearchApi(mockedTppBankSearchApi);
+
+        String xsrfToken = null;
+        {
+            final String user = "peter";
+            final String password = "1234";
+            log.info("DO Authorization ({}, {}) ==============================", user, password);
+            LoginBody loginBody = new LoginBody(user, password);
+            xsrfToken = auth(loginBody.username, loginBody.password);
+        }
+
+        String bankUUID = null;
+        {
+            final String keyword = "affe";
+            final Integer start = 1;
+            final Integer max = 2;
+            log.info("DO Bank Search ({}, {}, {}) ==============================", keyword, start, max);
+
+            when(mockedTppBankSearchApi.bankSearchGET(any(), any(), eq(keyword), eq(start), eq(max)))
+                    .thenReturn(gson.fromJson(readFile(getFilenameBankSearch(keyword, start, max)), BankSearchResponse.class));
+
+            bankUUID = bankSearch(keyword, start, max, xsrfToken);
+        }
+
+        List<String> services = null;
+        {
+            log.info("DO Bank Profile ({}) ============================== ", bankUUID);
+            when(mockedTppBankSearchApi.bankProfileGET(any(), any(), eq(bankUUID)))
+                    .thenReturn(gson.fromJson(readFile(getFilenameBankProfile(bankUUID)), BankProfileResponse.class));
+
+            services = bankProfile(xsrfToken, bankUUID);
+            Assertions.assertTrue(services.containsAll(Arrays.asList(new String[]{"List accounts", "List transactions", "Initiate payment"})));
```

Comment:
Can we have static import for Assertions

Reply:
done